xml/MAIN-SBP-Migrate-z-KVM.xml

```diff
@@ -116,7 +116,7 @@
         <para>It is recommended to use the SUSE-provided utilities that manage virtual machines
             using the libvirt API. The section <quote>Managing Virtual Machines with libvirt</quote>
             of the Virtualization Guide at <link
-                xlink:href="https://www.suse.com/documentation/sles-12/book_virt/data/part_virt_libvirt.html"
+                xlink:href="https://documentation.suse.com/sles/12-SP4/html/SLES-all/part-virt-libvirt.html"
```

Comment:
...and here...

Reply:
See above regarding MAIN-SBP-Migrate-z-KVM.xml